### ISSUE TITLE
Filter Illegal Header Fields

### ISSUE THREAD

**Issue by drothmaler** [FastEndpoints/FastEndpoints#615]
According to the [OpenApi Spcec](https://swagger.io/docs/specification/describing-parameters/#header-parameters) there are certain header parameters, that are not allowed as part of the regular parameters specification in the Swagger Spec.  These Headers (`Accept`, `Content-Type` and `Authorization`) are described using other OpenApi fields. 

The FastEndpoints Swagger generation should respect/filter that, when processing properties marked with `FromHeader`.

**Comment by dj-nitehawk** [comment]
it doesn't seem to be enforced for even minimal apis. just tried the following:
```cs
app.MapGet("/test", ([FromHeader] string accept) => Results.Ok(accept));
```
nswag generated a header parameter without any complains.

i pasted the generated swagger.json into https://editor.swagger.io/ 
no complains there either.

so i'm not sure if FE needs to worry about it if nobody else pays any attention to it...

swagger ui seems to just ignore any values entered in to these input boxes anyway.

**Comment by drothmaler** [comment]
Well I was going to create an Issues there too, just started here, because you generally try to make things better than the others :-)
And it would be pretty easy to implement in `OperationProcessor.Process`, shouldn't it? 

Some people/projects are pretty picky, when it comes to the formal correctness of specs. So I currently had to remove the headers in question from the typed Request object and access them by Context.Request instead...

**Comment by dj-nitehawk** [comment]
yeah i'll guess we should do it.
i'll add it when i get a chance...
if you'd like to do a PR, that would be fine too...

**Comment by KateyBee** [comment]
@drothmaler This has been implemented in `v5.23.0.8-beta`
